### DISCUSSION TITLE
Bytes and String of UTF8 in Unilang

### DISCUSSION THREAD

**Discussion by sharego** [linuxdeepin/developer-center#3429] [Features Request & Ideas | 特性请求 & 头脑风暴]
[Unilang](https://github.com/linuxdeepin/unilang), Is there a inner Bytes Object? How think about charset and unicode?

**Comment by BLumia** [comment]
Which project are you referring to?

**Reply by sharego** [reply]
Thanks,  [Unilang](https://github.com/linuxdeepin/unilang)

**Reply by BLumia** [reply]
cc @FrankHB
